src/components/app/list-new/AppList.tsx

```diff
@@ -901,66 +911,73 @@ export default function AppList() {
         )
     }
 
+    const renderFirstAppView = (): JSX.Element => {
+        return <GuidePage openDevtronAppCreateModel={openDevtronAppCreateModel} />
```

Comment:
Directly use <GuidePage /> instead of using an additional function to return the same.